Assets/Scripts/Entity/Player/PlayerState.cs

```diff
@@ -38,15 +38,6 @@ public class PlayerState : Damageable
 
         public static event PlayerIsDead OnPlayerDeath;
 
-        private void Awake()
-        {
-            // send event on initial values
-            OnPlayerHealthUpdate?.Invoke(health);
-            OnPlayerSaturationUpdate?.Invoke(saturation);
-            OnPlayerHydrationUpdate?.Invoke(hydration);
-            OnPlayerSanityUpdate?.Invoke(sanity);
-        }
```

Comment:
Please don't delete that, we need the events to be invoked on startup.

Reply:
nope, we do not - they only create errors as the events are invoked before everything has loaded correctly. On startup the default values (100 for all) are consistent with the UI. Changes will invoke the events again

Reply:
For the UI, yes but I put it there because other components needed it. What error do you mean? I didn't experience any

Reply:
Which other components?

Reply:
as sanity should be unaffected with all player states at 100, it does not need to get a notification on startup, where all values are at 100